include/swift/AST/DiagnosticsSIL.def

```diff
@@ -953,10 +953,10 @@ ERROR(regionbasedisolation_unknown_pattern, none,
 //
 
 ERROR(regionbasedisolation_transfer_yields_race_no_isolation, none,
-      "transferring value of non-Sendable type %0; later accesses could race",
+      "sending value of non-Sendable type %0; later accesses could race",
       (Type))
 ERROR(regionbasedisolation_transfer_yields_race_with_isolation, none,
-      "transferring value of non-Sendable type %0 from %1 context to %2 context; later accesses could race",
+      "sending value of non-Sendable type %0 from %1 context to %2 context; later accesses could race",
```

Comment:
Same here, what do you think of this?

`sending value of non-Sendable type %0 from %1 context to %2 context with later accesses risks a data race`

Reply:
This is a fallback path so I didn't wordsmith it as much. I agree your suggestion is better!